From 11894be1df53f84f50499429dacba52a10c02f08 Mon Sep 17 00:00:00 2001
From: Steven Dake <sdake@redhat.com>
Date: Mon, 24 Feb 2014 09:33:40 -0700
Subject: [PATCH] Remove pbr runtime dependency and replace with build time
 calculation

Requires some sed operation on REDHATOSCOLLECTCONFIGVERSION and
REDHATOSCOLLECTCONFIGRELEASE to runtime modify the package.
---
 os_collect_config/version.py | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/os_collect_config/version.py b/os_collect_config/version.py
index d8aec69..62a55ff 100644
--- a/os_collect_config/version.py
+++ b/os_collect_config/version.py
@@ -13,6 +13,15 @@
 #    under the License.
 
 
-import pbr.version
+class VersionInfo(object):
+    release = "REDHATOSCOLLECTCONFIGRELEASE"
+    version = "REDHATOSCOLLECTCONFIGVERSION"
 
-version_info = pbr.version.VersionInfo('os-collect-config')
+    def version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+version_info = VersionInfo()
+version_string = version_info.version_string
